src/Compilers/Core/Portable/PEWriter/PeWriter.cs

```diff
@@ -14,6 +14,13 @@
 
 namespace Microsoft.Cci
 {
+    internal sealed class PeWritingException : Exception
```

Comment:
It was, back when I added a resource and did the try..catch in PeWriter.cs. When I narrowed the scope and eliminated the resources I didn't move the exception.

I'd still like to keep it where it is since I think we want to fix this up ASAP to have its own resource string and to catch all possible PE writing exceptions.